evm_loader/faucet/Cargo.toml

```diff
@@ -1,21 +1,23 @@
 [package]
 name = "faucet"
-description = "Ethereum ERC20 Token Faucet Service"
+description = "NeonLabs Faucet Service"
 authors = ["NeonLabs Maintainers <maintainers@neonlabs.org>"]
 repository = "https://github.com/neonlabsorg/neon-evm/evm_loader/faucet"
-version = "0.5.4-dev"
+version = "0.6.0-dev"
```

Comment:
This field is changed after delivering a new release version. So please revert to the 0.5.4-dev

Reply:
No problem, but it's another version after all. Would 0.5.5-dev do?

Reply:
If you like to distinguish the save version please add the commit like here: https://github.com/neonlabsorg/neon-evm/blob/bfad6460a1fd1a88eaa47af53d8f3a7f0ffbaa03/evm_loader/cli/src/main.rs#L1436

Reply:
Well, let it be old version. Who cares?